### ISSUE TITLE
Arcade's type hinting is not available to modules that import arcade

### ISSUE THREAD

**Issue by SirGnip** [pythonarcade/arcade#781]
Running mypy on a project that uses arcade doesn't seem to be able to make use of the 2222type hints in arcade:

(repro done on Windows 10 with GitBash)

```
$ python -m venv venv
$ source venv/Scripts/activate
$ pip install arcade==2.4.3

# create mygame.py
import arcade
print(arcade.Sprite())

$ python mygame.py
<arcade.sprite.Sprite object at 0x000001C495549470>

$ pip install mypy
$ mypy mygame.py
mygame.py:1: error: Skipping analyzing 'arcade': found module but no type hints or library stubs
mygame.py:1: note: See https://mypy.readthedocs.io/en/latest/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
```

If you run mypy on the arcade repo itself, it does the type hinting as expected.  But, if you run mypy on a project that imports arcade, you get failures.  Arcade's type hinting was originally exposed to other packages via https://github.com/pythonarcade/arcade/issues/533.  I'm guessing tthe regression was introduced because the `py.typed` file was dropped during this commit:  https://github.com/pythonarcade/arcade/commit/f0ff23831e90180685ee13e88bb6b5adb8b2c379#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7.

**Comment by Cleptomania** [comment]
Planning to fix this as part of https://github.com/pythonarcade/arcade/pull/780. Will update with the specific commit when it's done.

**Comment by Cleptomania** [comment]
This is fixed in https://github.com/pythonarcade/arcade/pull/780/commits/c82d868aada644dfccdf712032c147cc3452515e

**Comment by Cleptomania** [comment]
This is fixed as of the 2.5 release by https://github.com/pythonarcade/arcade/pull/780
